packages/runtime/test-runtime-utils/src/mocksDataStoreContext.ts

```diff
@@ -75,6 +77,8 @@ export class MockFluidDataStoreContext implements IFluidDataStoreContext {
 	public createProps?: any;
 	public scope: FluidObject = undefined as any;
 
+	public minVersionForCollab: MinimumVersionForCollab = defaultMinVersionForCollab;
```

Comment:
Since we have the getter, should this be public too? Do we have scenarios that set it?